.github/workflows/component-bumper-release-0.42.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     - name: Set environment variables
       run: echo "BASE_BRANCH=release-0.42" >> $GITHUB_ENV
 
+    - name: Login to Quay
+      run: docker login -u="kubevirt+network" -p="${{ secrets.QUAY_ROBOT_TOKEN }}" quay.io
```

Comment:
Are we allowed to store those tokens here ?

Reply:
why not? the secret is stored in the github repo secret section, and securely used here.

Reply:
fair enough